solr-search/src/main/java/edu/unc/lib/dl/search/solr/service/NeighborQueryService.java

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.dl.search.solr.service;
+
+import static edu.unc.lib.dl.search.solr.util.SearchFieldKeys.RESOURCE_TYPE;
+import static edu.unc.lib.dl.search.solr.util.SearchFieldKeys.TITLE_LC;
+import static edu.unc.lib.dl.search.solr.util.SolrSettings.sanitize;
+import static java.lang.String.format;
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.jena.ext.com.google.common.collect.Lists;
+import org.apache.solr.client.solrj.SolrQuery;
+import org.apache.solr.client.solrj.SolrServerException;
+import org.apache.solr.client.solrj.response.QueryResponse;
+
+import edu.unc.lib.dl.acl.util.AccessGroupSet;
+import edu.unc.lib.dl.search.solr.exception.SolrRuntimeException;
+import edu.unc.lib.dl.search.solr.model.BriefObjectMetadataBean;
+import edu.unc.lib.dl.search.solr.model.CutoffFacet;
+import edu.unc.lib.dl.search.solr.util.FacetFieldUtil;
+import edu.unc.lib.dl.search.solr.util.SearchFieldKeys;
+import edu.unc.lib.dl.util.ResourceType;
+
+/**
+ * Service that retrieves objects neighboring a selected object.
+ *
+ * @author bbpennel
+ *
+ */
+public class NeighborQueryService extends AbstractQueryService {
+
+    protected FacetFieldUtil facetFieldUtil;
+
+    /**
+     * Retrieves a list of the closest windowSize neighbors within the parent container of the specified object,
+     * using title sort order. The first windowSize / 2 - 1 neighbors are retrieved to each side
```

Comment:
Am I correct in thinking you meant to say "...(windowSize -1) / 2 neighbors are retrieved..." Whenever windowSize is even, windowSize - 2 neighbors will be retrieved; otherwise, windowSize - 1 neighbors will be retrieved.

Reply:
Hmm, yeah I think that's right, I'll fix it